OPHD/States/MapViewState.cpp

```diff
@@ -206,7 +206,7 @@ void MapViewState::initialize()
 	MAIN_FONT = &fontCache.load(constants::FONT_PRIMARY, constants::FontPrimaryNormal);
 
 	delete mPathSolver;
-	mPathSolver = new micropather::MicroPather(mTileMap);
+	mPathSolver = new micropather::MicroPather(mTileMap, 250, 6, false);
```

Comment:
Hmm, that's not very self documenting at the point of call. I needed to look up what those values meant:
```cpp
MicroPather(Graph* graph, std::size_t allocate = 250, std::size_t typicalAdjacent = 6, bool cache = true);
```

Considering MicroPather is an external library (which I don't much want to maintain or have our local copy diverge from the original), I'm not sure there's really much to do about it. Though I do wonder if we could somehow wrap or cordon off uses of the library so we don't have multiple calls like this with non-obvious parameter values.

Reply:
There isn't. I just took the default values right out of the pathing libary.

I'm really not worried about wrapping things to hide details. That just makes things more complex than necessary just to make the code less obtuse. A comment here stating that these are the library provided defaults may be more useful though the pather is only instantiated during loading and when starting a new game... I don't see where we'd need additional uses of it.